src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Client.kt

```diff
@@ -50,12 +54,21 @@ data class Client(
   @Column(length = 2000)
   @Convert(converter = TokenSettingsConverter::class)
   var tokenSettings: TokenSettings,
+
+  @OneToMany
+  @JoinColumn(name = "registeredClientId")
+  @Where(clause = "access_token_issued_at =(select max(oa.access_token_issued_at) from oauth2_authorization oa)")
```

Comment:
Probably safest to include filter on `authorization_grant_type` in the where clause

Reply:
Added